rules/system-health.mk

```diff
@@ -0,0 +1,9 @@
+# system health python2 wheel
```

Comment:
Can you please also add a `system-health.dep` file for our new package caching system? You should be able to easily copy an existing file for another whl, like https://github.com/Azure/sonic-buildimage/blob/master/rules/sonic_bgpcfgd.dep and make minor changes.

Reply:
Sure.